### PR TITLE
Docs: Added Kotlin Math cbrt function

### LINKED ISSUE #3519: [Term Entry] Kotlin Math .cbrt()
### Reason/inspiration (optional)

We would like a new term entry on the `.cbrt()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by VividhPandey003**
### Description

Closes #3519 

### Type of Change

- Updating the documentation

### Checklist

- [+] All writings are my own.
- [+] My entry follows the Codecademy Docs style guide.
- [+] My changes generate no new warnings.
- [+] I have performed a self-review of my own writing and code.
- [+] I have checked my entry and corrected any misspellings.
- [+] I have made corresponding changes to the documentation if needed.
- [+] I have confirmed my changes are not being pushed from my forked `main` branch.
- [+] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [+] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+The **`Math.cbrt()`** method returns the cube root of the argument.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:13]:
```suggestion

The **`math.cbrt()`** method returns the cube root of the given argument.
```

File: content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Syntax
+
+```pseudo
+Math.cbrt(n)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:18]:
```suggestion
cbrt(n)
```

Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:18]: there should also be a brief sentence explaining `n`

File: content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+## Example
+The following example uses `Math.cbrt()` to return the cube roots of `-6.5`, `2`, and `-340.8`:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:22]:
```suggestion

The following example uses `cbrt()` to return the cube roots of `-6.5`, `2`, and `-340.8`:
```

Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:22]: there should always be an empty line after a header

File: content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+Title: '.cbrt()'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/cbrt/cbrt.md:2]:
```suggestion
Title: 'cbrt()'
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@VividhPandey003 I've added my initial comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by VividhPandey003** [comment]
@caupolicandiaz I have made the required changes. Could you please review it once.
Thanks

**Review by caupolicandiaz — APPROVED** [review]
@VividhPandey003 I just made a couple of additional direct edits. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@VividhPandey003 @caupolicandiaz Looks good.

**Pull request merged**